travis-build/crypt-common.sh

```diff
@@ -6,12 +6,12 @@ PROJECT=professionalserviceslabs
 KEYRING=ps-keyring
 KEY=commercetools-paypal-plus-integration
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
 CONFIG_DIR="${SCRIPT_DIR}/configuration"
 PLAIN_FILE="${CONFIG_DIR}/travis-build-settings.sh"
 CIPHER_FILE="${CONFIG_DIR}/travis-build-settings.sh.enc"
 
-function encryptDecrypt() {
+encryptDecrypt() {
```

Comment:
but this should work also with `function`, right?

Reply:
yes, in all POSIX compatible shells (`bash`, `csh`, `dash`, `zsh` and so on)